pages/common/piper.md

```diff
@@ -0,0 +1,25 @@
+# piper
+
+> A fast, local neural text to speech system.
+> Try out and download speech models from <https://rhasspy.github.io/piper-samples>.
+> More information: <https://github.com/rhasspy/piper>.
+
+- Output a WAV [f]ile using a text-to-speech [m]odel (assuming config json at path to model_path + .json):
```

Comment:
Hi, IG it would be better to rephrase this `(assuming config json at path to model_path + .json)`

```suggestion
- Output a WAV [f]ile using a text-to-speech [m]odel with `config.json` at the model path:
```

Reply:
The original is definitely clunky, but this suggestion would be incorrect. If you have a model at `long/path/foo.onnx`, then it will by default look at `long/path/foo.onnx.json`, not `config.json`. Otherwise you need to specify the JSON file using `-c`.

The main reason of mentioning this at all, is that while the files are named correctly on huggingface (i.e. each `foo.onnx` is accompanied by a `foo.onnx.json`), it seems if you follow the standard way of downloading the files by the links given the JSON name gets mangled into being something like `foo_foo.onnx_foo.json` which breaks this automatic detection, unless you manually CURL'd the URLs yourself.

Reply:
would it perhaps be ok if I simply reword `assuming config json` in the original to `assuming a config file`?

Reply:
> would it perhaps be ok if I simply reword `assuming config json` in the original to `assuming a config file`?

Yeah, that would work too.